temba/channels/models.py

```diff
@@ -1329,6 +1330,12 @@ def success(cls, channel, msg, msg_status, start, external_id=None, event=None,
                                       response_status=event.status_code,
                                       request_time=request_time_ms)
 
+            # Send data to Chatbase API
+            if hasattr(msg, 'is_org_connected_to_chatbase'):
+                Org.register_chatbase_log(api_key=msg.chatbase_api_key, version=msg.chatbase_version, org_id=msg.org,
```

Comment:
We don't need to store the chatbase credentials on the message or the log - we can get these from the org when sending the logs